execution-engine/contracts/hdac-system/pop/src/pop_impl/mod.rs

```diff
@@ -385,19 +387,26 @@ impl ProofOfProfessionContract {
         let mut commissions = ContractClaim::read_commission()?;
         let mut rewards = ContractClaim::read_reward()?;
 
+        // We should calculate how much time passed from last distribute
+        let current_blocktime: u64 = runtime::get_blocktime().into();
+        let last_distribute_time: u64 = ContractClaim::read_last_distribute()?;
+        let time_passed: u64 = if current_blocktime <= last_distribute_time {
+            0
+        } else {
+            current_blocktime - last_distribute_time
```

Comment:
How about using BlockTime? BlockTime has saturating_sub method which prevents (over,under)flow.

Reply:
At first I used BlockTime and saturating_sub, but I changed because using u64 could reduce memory space, I thought.

Reply:
I think every abstraction takes some costs. But you think it is fairly worthwhile, I'll accept.